apps/dashboard/lib/rclone_util.rb

```diff
@@ -0,0 +1,147 @@
+require "json"
+require "open3"
+require "pathname"
+
+class RcloneUtil
```

Comment:
I'm not sure if I see the value in this utility file. I mean, why not just have the all the logic in `remove_file.rb`? I'm not particularly worried about the size of any 1 file.

Reply:
My main reason for that is that once RemoteTransfer is implemented it would also use some of the functions that are also used currently by RemoteFile. So for example once the rclone path or config is made configurable there would be less to change as it is all in one place.

Reply:
Got it, thx.